### PR TITLE
[Accepted] Opened on openapi-generator

### PR DESCRIPTION
**Pull request by xiedaxia1hao**
In this PR, I fixed some newly found flaky tests for the repo `openapi-generator` and opened a PR in its original repo.

**Update: `openapi-generator` just accepted my [PR](https://github.com/OpenAPITools/openapi-generator/pull/10485).**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1259,6 +1259,11 @@ https://github.com/odrotbohm/sos,d4ed68a143b637c7e2d523edad1b105bf6c2d996,20-mod
...
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNewLineLinux,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNewLineWindows,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsMixed,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNoDuplicate,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
```
Comment (winglam) [inline pr-data.csv:1266]: It is great that the developer agreed to merge the changes (but has yet to do so).

Can you please ensure that the build passes with your changes?
https://github.com/TestingResearchIllinois/idoft/pull/81/checks?check_run_id=3728028641

(E.g., one problem seems to be `org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNoDuplicate` should be `org.openapitools.codegen.java.AbstractJavaCodegenTest.testAdditionalModelTypeAnnotationsNoDuplicate`)

Comment (xiedaxia1hao) [reply pr-data.csv:1266]: Thank you pointing it out! Will look into this now.

Comment (xiedaxia1hao) [reply pr-data.csv]: Interesting.. After changing the format, the `format-checker` still failed. I double-checked for several times but cannot figure out why my pr still violate the format.. 😅

Comment (winglam) [reply pr-data.csv]: Thanks for your changes.

> ERROR: On file pr-data.csv: The file is not properly ordered
https://github.com/TestingResearchIllinois/idoft/pull/81/checks?check_run_id=3728150389#step:6:8

You can sort the file with the command in the following section of the readme: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test

Comment (xiedaxia1hao) [reply pr-data.csv]: Sorry for not getting back to you sooner. Just sort the .csv file based on the document you shared and it works! This PR looks good now : )

**Comment by winglam** [comment]
LGTM. Thanks for your changes!

**Pull request merged**